assets/sass/bootstrap/_timeline.scss

```
@@ -0,0 +1,34 @@
+//
+// Base styles
+//
+
+ul.timeline {
+	list-style-type: none;
+	position: relative;
+	&:before {
+		content: ' ';
+		background: #d4d9df;
```

You can use one of the theme's color variables here.

```suggestion
		background: $info;
```